src/scenes/home/header/topNav/topNav.css

```diff
@@ -1,11 +1,28 @@
 .topNav {
   display: flex;
-  justify-content: space-between;
+  justify-content: flex-end;
   flex: 1 60%;
-  margin-right:5vw;
```

Comment:
Removing the margin-right from .topNav pushes the topNav links far to the right, inconsistent with the logo text on the left (which has a margin-left) - this gives a horizontally uncentered feel to the layout:
![screenclip](https://cloud.githubusercontent.com/assets/8835055/26531753/03b39c80-43be-11e7-9ee6-6b32fda3a2c4.png)

If we're going to remove the `margin-right:5vw` from .topNav in topNav.css, then the `margin-left: 80px` from .logo in logo.css should also be removed.  We could then add something like `margin: 0 80px` to the class .header in header.css, to create equal margin on the left and right sides of the Header, eliminating the need for left/right margin rules in the CSS for the TopNav or Logo components.